Assets/Scripts/ProgressManager.cs

```diff
@@ -14,6 +14,9 @@ public class ProgressManager: Singleton<ProgressManager>
     public List<Collectible> stampCollectibles;                        // List of collectibles needed to be able to add stamps in the inspector
     public Dictionary<Collectible, bool> stampsUnlockStatus;           // Use a dictinoary to keep track of what stamps are unlocked
 
+    public List<Collectible> bingoBallCollectibles;                    // List of collectibles needed to be able to add bingo balls in the inspector
+    public Dictionary<Collectible, bool> bingoBallsUnlockStatus;        // Keeps track of unlock status for bingo balls

```

Comment:
Should these actually be using `BingoBallCollectible` instead? Maybe for stamps too they should be using their own collectible type...